spec/views/admin/finance/statements/index.html.erb_spec.rb

```diff
@@ -24,7 +24,11 @@
   it 'has a link to each statement' do
     render(locals:)
 
-    raw_statements.each { |s| expect(rendered).to have_link('View', href: admin_finance_statement_path(s)) }
+    presented_statements = Admin::StatementPresenter.wrap(raw_statements)
+    presented_statements.each do |s|
+      link_text = "View statement for #{s.lead_provider_name} in #{s.month_and_year}"
+      expect(rendered).to have_link(link_text, href: admin_finance_statement_path(s))
+    end
```

Comment:
A little wordier but maybe something along the lines of this?

```suggestion    
    raw_statements.each do |s|
      statement_date = "#{Date::MONTHNAMES.fetch(s.month)} #{s.year}"

      expect(rendered).to have_link("View statement for #{s.lead_provider.name} in #{statement_date}", href: admin_finance_statement_path(s))
    end
```

Reply:
Perhaps moving the statement date building out into its own class would make sense? Then it can be used everywhere while being fairly concise.

Reply:
That doesn't need to happen now though, this is still an improvement over what was there.